packages/phone-number-privacy/combiner/src/pnp/services/logSignerResponses.ts

```diff
@@ -106,4 +107,24 @@ export class PnpDiscrepanciesLogger {
       session.warnings.push(WarningMessage.INCONSISTENT_SIGNER_QUERY_MEASUREMENTS)
     }
   }
+
+  logFailOpenResponses(session: Session<PnpQuotaRequest> | Session<SignMessageRequest>): void {
+    session.responses.forEach((response) => {
```

Comment:
do you think it's worth it to push one (or multiple) FAILING_OPEN errors to the `session.warnings` when logging these discrepancies? I am on the fence here: would be slightly more consistent with the discrepancy logging, but also not sure if we necessarily want to make it super obvious when ODIS is failing open?

Reply:
Yea, I don't think we should return the FAILING_OPEN warning to the client. The important thing is that we log it for ourselves